code/src/nuvla/connector/nuvlabox_connector.py

```diff
@@ -135,10 +165,118 @@ def start(self, **kwargs):
     def stop(self, **kwargs):
         pass
 
+    @should_connect
+    def update_nuvlabox_engine(self, **kwargs):
+        self.job.set_progress(10)
+
+        # 1st - get the NuvlaBox Compute API endpoint and credentials
+        self.setup_ssl_credentials()
+
+        self.job.set_progress(50)
+
+        # 2nd - set the Docker args
+        # image name
+        logging.info('Preparing parameters for NuvlaBox update')
+        image = 'nuvlabox/installer:master'
+        detach = True
+
+        # container name
+        container_name = f'installer' #-{self.job.id.replace("/", "-")}'
```

Comment:
Is the comment expected to stay in the release ?

Reply:
up to you :p I can remove them if you want

Reply:
If it's there for a future use you it can stay but if it's an old code let there by mistake, it's better to remove it.